Source/ACE/Entity/Player.cs

```diff
@@ -2782,6 +2830,17 @@ public void DoMoveTo(WorldObject wo)
             moveToObjectChain.EnqueueChain();
         }
 
+        public void DoCollideWith(WorldObject wo)
```

Comment:
the purpose is to force the player to run into or collide with the portal. Portal's use radius is -0.1. So it needs a MoveToPosition command not a MoveToObject command.